Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -39,8 +84,19 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```

Comment:
What does RemoveFirst and RemoveAt have in common :) ?

Reply:
"If an array is empty or null, returns input array."
This is what I've should replaced within a function.. But is there anything else?

Reply:
NullOrEmptyArray function